docs/setting-up.md

```diff
@@ -75,3 +72,7 @@ We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA (exp
 Support requests related to other IDEs will not be entertained.
 
 Refer to [this document](ide-usage.md) if you wish to set up an IDE for developing TEAMMATES.
+
+## Step 5: Start Developing
```

Comment:
Avoid capitalising `developing` for style consistency with the previous 4 steps.